include/circt/Dialect/FIRRTL/FIRRTLUtils.h

```diff
@@ -33,6 +33,9 @@ IntegerAttr getIntZerosAttr(Type type);
 /// Utility for generating a constant all ones attribute.
 IntegerAttr getIntOnesAttr(Type type);
 
+/// Return the single assignment to a Property value.
+PropAssignOp getPropertyAssignment(Value value);
```

Comment:
Would love to combine/share this across connect-likes, and use for references (`RefDefineOp`), but for now this looks good.

Reply:
Yeah, we can definitely share the core logic here.